config/application.example.yml

```diff
@@ -6,7 +6,8 @@ HTTP_PASS: password
 
 # External dependencies
 DATABASE_URL: postgres://postgres@localhost:5432/tvs_development?template=template0&pool=5&encoding=unicode
-REDIS_URL: redis://localhost:6379
+REDIS_CACHE_URL: redis://localhost:6379
+REDIS_QUEUE_URL: redis://localhost:6379
```

Comment:
A good change as it should allow the app to start 👍 

I guess it will work for everything to connect to the same Reis instance however worth noting the loss of parity. With docker-compose and on the live environments there are 2 separate instances. We'd probably need out of docker instructions for spinning up 2 redis instance instances on different ports.